reference/configuration/framework.rst

```diff
@@ -2483,6 +2489,31 @@ annotation changes). For performance reasons, it is recommended to disable
 debug mode in production, which will happen automatically if you use the
 default value.
 
+
+secrets
+~~~~~~~
+
+decryption_env_var
+..................
+
+**type**: ``string`` **default**: ``base64:default::SYMFONY_DECRYPTION_SECRET``
+
+The environment variable that contains the decryption key.
+
+local_dotenv_file
+.................
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/.env.%kernel.environment%.local``
+
+Path to an dotenv file that holds secrets. This is primarily used for testing.
+
+vault_directory
+...............
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/config/secrets/%kernel.environment%``
+
+The directory where the secrets is stored.
```

Comment:
```suggestion
The directory where the secrets file is stored.
```

Reply:
Hm.. It is just not one file. I updated this line

Reply:
Perfect 👌🏻